client/block.c

```diff
@@ -1783,13 +1782,11 @@ int xdag_get_transactions(xdag_hash_t hash, void *data, int (*callback)(void*, i
 	for (i = 0; i < n; ++i) {
 		if (!i || block_array[i] != block_array[i - 1]) {
 			struct block_internal *ri = block_array[i];
-			if (ri->flags & BI_APPLIED) {
```

Comment:
Is this condition not necessary?

Reply:
Not necessary, to query transactions users need to know the status of the tx block even the block is pending or rejected.